src/client/context.rs

```diff
@@ -714,13 +748,49 @@ impl Context {
         rest::edit_emoji(guild_id.into().0, emoji_id.into().0, map)
     }
 
+    /// Allows to configure specific guild options like icon, AFK channel, etc.
+    ///
+    /// You can see available methods in [`EditGuild`] docs.
+    ///
+    /// # Examples
+    ///
+    /// Change server's icon using file icon.png:
+    ///
+    /// ```rust,ignore
+    /// use serenity::utils;
+    ///
+    /// // We are using read_image helper function from utils.
+    /// let base64_icon = match utils::read_image("./icon.png") {
+    ///     Ok(base64_icon) => base64_icon,
+    ///     Err(why) => {
+    ///         println!("Error reading image: {:?}", why);
+    ///         return;
+    ///     },
+    /// };
+    ///
+    /// context.edit_guild(guild_id, |g|
+    ///     g.icon(base64_icon));
+    /// ```
     pub fn edit_guild<F, G>(&self, guild_id: G, f: F) -> Result<PartialGuild>
         where F: FnOnce(EditGuild) -> EditGuild, G: Into<GuildId> {
         let map = f(EditGuild::default()).0.build();
 
         rest::edit_guild(guild_id.into().0, map)
     }
 
+    /// Allows to do specific things with members of a server
+    /// like mute, change nickname, etc.
+    /// Full list of methods is available at [`EditMember`] docs.
+    ///
+    /// # Examples
+    ///
+    /// Mute a member and set their roles to just one role with
+    /// predefined Id.
+    /// ```rust,ignore
+    /// context.edit_member(guild_id, user_id, |m|
+    ///     m.mute()
```

Comment:
`EditMember::mute` takes a bool, so let's do `mute(true)`, and put the `m` after the `|m|` closure opener above